src/client/pps.go

```diff
@@ -709,13 +709,17 @@ func (c APIClient) ListPipelineHistory(pipeline string, history int64) ([]*pps.P
 }
 
 // DeletePipeline deletes a pipeline along with its output Repo.
-func (c APIClient) DeletePipeline(name string, force bool) error {
+func (c APIClient) DeletePipeline(name string, force bool, splitTransaction ...bool) error {
```

Comment:
One note here: We are using a variatic function here for DeletePipeline() to avoid breaking user code that uses the go client. A better pattern for this would be functional options, which may be a better way of doing this in the future, but it is out of scope for this change.